db/models/reviews.models.js

```diff
@@ -15,3 +15,14 @@ exports.fetchReviewById = (id) => {
       }
     });
 };
+exports.updateReviewVotes = (id, votesToAdd) => {
+  return this.fetchReviewById(id).then(({ votes }) => {
```

Comment:
You don't need to make two requests to the database. You can perform the updating of the votes using only SQL. This was we don't have multple requests to a database for a request to a single endpoint.